torch/_inductor/kernel/mm.py

```diff
@@ -142,6 +142,20 @@ def tuned_mm(mat1, mat2, *, layout=None):
             )
         log.debug("Added %d cutlass gemm configs.", len(ops))
 
+    from torch._inductor.ir import FixedLayout, FlexibleLayout
+
+    if (
+        len(choices) == 1
+        and use_aten_gemm_kernels()
```

Comment:
I guess it's probably a bit cleaner to just remove this condition?

Reply:
Do you mean remove the use_aten_gemm_kernels? We would also remove the restriction above then, right? If it safe to do so, i can remove both

Reply:
The idea is that if you only have one kernel choice, then the layout choice won't have any affect on which kernel gets chosen. So it's less likely to matter as much.